### DISCUSSION TITLE
Jellyfin web page no longer starts on boot after setting up nfs?

### DISCUSSION THREAD

**Discussion by LoadingStill** [tteck/Proxmox#2985] [Help Needed] [answered]
Good evening,

I have installed Jellyfin using the scripts today using a privileged container.  On first start up and reboot, everything is fine and good.
But when I install nfs-common and edit the fstab to point to the NAS and restart to make sure on boot it mounts, Jellyfin no longer can be accessed from the browser.
But systemctl status shows:
```
root@jellyfin:~# systemctl status jellyfin.service 
* jellyfin.service - Jellyfin Media Server
     Loaded: loaded (/usr/lib/systemd/system/jellyfin.service; enabled; preset: enabled)
    Drop-In: /etc/systemd/system/jellyfin.service.d
             `-jellyfin.service.conf
     Active: active (running) since Tue 2024-05-14 21:25:51 EDT; 27s ago
   Main PID: 241 (jellyfin)
      Tasks: 18 (limit: 9166)
     Memory: 278.3M (peak: 278.6M)
        CPU: 3.751s
     CGroup: /system.slice/jellyfin.service
             `-241 /usr/bin/jellyfin --webdir=/usr/share/jellyfin/web --ffmpeg=/usr/lib/jellyfin-ffmpeg/ffmpeg
 ```

If I `systemctl restart jellyfin` the website loads properly and the systemctl status stays the same, just a different running time.
 
 Any ideas where I should look to resolve this issue, at this point I am stumped.
 
System Info:
* privileged LXC container
* Ubuntu 24.04 LTS Jellyfin lxc/tty1
* May 14, 2024, installation date
* Jellyfin 10.

Side question, I do not have this issue on any config changes under an unprivileged LXC container.  Is there a way to mount NFS or CIFS to an unprivileged container?

I did not see this topic listed under previous discussions.  If I missed it, please link it, so I may read that page.  Thank you.

**Comment by tteck** [comment, marked as answer]
I haven't even checked if hardware acceleration functions properly on Ubuntu 24.04 yet.

**Reply by LoadingStill** [reply]
Are you saying nfs require hardware acceleration to work? And that could be a reason for my issue?

I am new to lxc containers in general and still am trying to learn.

**Reply by LoadingStill** [reply]
I am also just now seeing I used 24.04 and not 22.04 per the scripts default.
I will change that when I can tonight.
